external/fv3fit/fv3fit/reservoir/transformers/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .transformer import Transformer
```

Comment:
I would consider removing `Transformer` at the top level if `ReloadableTransformer` is the public facing type used elsewhere.  I do understand that this is the more general type, but it might be confusing why `ReloadableTransformer` is when they all appear to be `Transformer` types.